linchpin/InventoryFilters/JSONInventoryFormatter.py

```diff
@@ -85,11 +85,13 @@ def add_common_vars(self, host_groups, layout):
                 self.config["_meta"]["hostvars"] = {}
             for item in items:
                 for var in cvrs:
-                    self.config["_meta"]["hostvars"][item] = {}
-                    if cvrs[var] == "__IP__":
```

Comment:
We don't need this special case anymore?

Reply:
No, if `__IP__` isn't defined in the config it's added with a default value, so this case will be handled in the for loop below (`for cfg_item in config:`)